dev/bots/prepare_package.dart

```diff
@@ -2,15 +2,20 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
+import 'dart:typed_data';
 
 import 'package:args/args.dart';
 import 'package:path/path.dart' as path;
+import 'package:http/http.dart' as http;
 
 const String CHROMIUM_REPO =
     'https://chromium.googlesource.com/external/github.com/flutter/flutter';
 const String GITHUB_REPO = 'https://github.com/flutter/flutter.git';
+const String MINGIT_URL = 'https://storage.googleapis.com/flutter_infra/mingit/'
```

Comment:
maybe MINGIT_WIN_URL to emphasize that this is win only?

Reply:
Done.